source/renderer/app/components/wallet/paper-wallet-certificate/CompletionDialog.js

```diff
@@ -168,6 +169,7 @@ export default class CompletionDialog extends Component<Props, State> {
                 aria-hidden
               >
                 {cardanoExplorerLink}
+                <SVGInline svg={externalLinkIcon} />
```

Comment:
Shouldn't it be underlined only on hovering?

Reply:
cc @a-rukin 

![image](https://user-images.githubusercontent.com/1504716/60741751-09f12500-9f41-11e9-843a-96113439587b.png)